test/d2l-search-results-count.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+		<title>d2l-search-results-count test</title>
+		<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../../web-component-tester/browser.js"></script>
+		<script src="../../iron-test-helpers/mock-interactions.js"></script>
+		<script src="../../iron-test-helpers/test-helpers.js"></script>
+		<link rel="import" href="../all-imports.html">
```

Comment:
Just to keep the import consistent with the [demo file](https://github.com/BrightspaceUI/facet-filter-sort/pull/3/files#diff-8530dd38feb4e1a55635714f592cf176R11)

```suggestion
		<link rel="import" href="../d2l-search-results-count.html">
```

Or change both to all-imports